netci.groovy

```diff
@@ -3589,6 +3692,9 @@ def static shouldGenerateFlowJob(def scenario, def isPR, def architecture, def c
             return false
         }
     }
+    else if (isCrossGenComparisonScenario(scenario)) {
+        return shouldGenerateCrossGenComparisonJob(os, architecture, configuration, scenario)
```

Comment:
I'd prefer this logic be inlined below, as is done for all other scenarios. Same in the other place this is used. But I'm ok with this way, also.